<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Active project info</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>891</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="sceneLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="project_menu">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="project_menu"/>
  </widget>
  <widget class="QToolBar" name="featuresToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExtract_Features"/>
   <addaction name="actionMatch_Features"/>
   <addaction name="actionStart_Reconstruction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="visualizationToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVisualizeBinary"/>
   <addaction name="actionRunExampleReconstruction"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSearch_Image"/>
   <addaction name="actionBuildToBinary"/>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>Create New Project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionBuildToBinary">
   <property name="text">
    <string>Build into a binary file</string>
   </property>
  </action>
  <action name="actionExtract_Features">
   <property name="text">
    <string>Extract Features</string>
   </property>
   <property name="toolTip">
    <string>Extract Features</string>
   </property>
  </action>
  <action name="actionMatch_Features">
   <property name="text">
    <string>Match Features</string>
   </property>
   <property name="toolTip">
    <string>Match Features</string>
   </property>
  </action>
  <action name="actionStart_Reconstruction">
   <property name="text">
    <string>Start Reconstruction</string>
   </property>
   <property name="toolTip">
    <string>Start Recostruction</string>
   </property>
  </action>
  <action name="actionSearch_Image">
   <property name="text">
    <string>Search Image</string>
   </property>
   <property name="toolTip">
    <string>Search Image</string>
   </property>
  </action>
  <action name="actionVisualizeBinary">
   <property name="text">
    <string>Visualize a binary (default - out/model-0.binary)</string>
   </property>
   <property name="toolTip">
    <string>Visualize a binary (default - out/model-0.binary)</string>
   </property>
  </action>
  <action name="actionRunExampleReconstruction">
   <property name="text">
    <string>Run Theia's view_reconstruction binary</string>
   </property>
   <property name="toolTip">
    <string>Run Theia's view_reconstruction binary</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
